packages/editor/src/store/controls.js

```diff
@@ -23,6 +23,38 @@ export function getRegistry() {
 	return { type: 'GET_REGISTRY' };
 }
 
+/**
+ * Function returning a sessionStorage key to set or retrieve a given post's
+ * automatic session backup.
+ *
+ * Keys are crucially prefixed with 'wp-autosave-' so that wp-login.php's
+ * `loggedout` handler can clear sessionStorage of any user-private content.
+ *
+ * @see https://github.com/WordPress/wordpress-develop/blob/6dad32d2aed47e6c0cf2aee8410645f6d7aba6bd/src/wp-login.php#L103
+ *
+ * @param {string} postId  Post ID.
+ * @return {string}        sessionStorage key
+ */
+function postKey( postId ) {
+	return `wp-autosave-block-editor-post-${ postId }`;
+}
+
+export function localAutosaveGet( postId ) {
+	return window.sessionStorage.getItem( postKey( postId ) );
+}
+
+export function localAutosaveSet( postId, title, content, excerpt ) {
+	window.sessionStorage.setItem( postKey( postId ), JSON.stringify( {
+		post_title: title,
+		content,
+		excerpt,
+	} ) );
+}
+
+export function localAutosaveClear( postId ) {
+	window.sessionStorage.removeItem( postKey( postId ) );
+}
```

Comment:
What's the reasoning for keeping these utils here and using a control. I do wonder if we should just use these functions directly from like a `autosave-utils` or something like that?

Reply:
I went for controls to see how it would feel to be able to dispatch saving actions just like with regular autosave.

> `autosave-utils`

These would have to be available in E2E tests, so presumably they would be globally available. Is it worth adding a new module just for this?

I guess controls are flexible for both good and bad. What's your preference here?

Reply:
I think since it has no relation with state, I'd personally prefer just a utility file somewhere